Fable.Remoting.Client/Proxy.fs

```diff
@@ -178,7 +178,11 @@ module Proxy =
                     match inputArgumentTypes.Length with
                     | 1 when not (Convert.arrayLike inputArgumentTypes.[0]) ->
                         let typeInfo = TypeInfo.Tuple(fun _ -> inputArgumentTypes)
-                        let requestBodyJson = Convert.serialize inputArguments.[0] typeInfo
+                        let requestBodyJson =
+                            inputArguments
+                            |> Array.tryHead
+                            |> Option.map (fun arg -> Convert.serialize arg typeInfo)
+                            |> Option.defaultValue ""
```

Comment:
Default value should probably be a valid JSON in any case, maybe `"{}"` or `"[]"` could do

Reply:
Hi @Zaid-Ajaj tbh what I worked out for this branch of code as it is a GET request that it just discards this arg anyway